src/nanodbc.cpp

```diff
@@ -2912,6 +2951,22 @@ inline void result::result_impl::get_ref_impl<string_type>(short column, string_
             return;
         }
 
+        case SQL_C_TIME:
+        {
+            const time t = *reinterpret_cast<time*>(col.pdata_ + rowset_position_ * col.clen_);
+            std::tm st = { 0 };
+            st.tm_hour = t.hour;
+            st.tm_min = t.min;
+            st.tm_sec = t.sec;
+            char* old_lc_time = std::setlocale(LC_TIME, NULL);
```

Comment:
Since 3cdc1202019eb8c25f43249ddac1bd6c112b69a2, I have made `NULL` no longer kosher and it should read `nullptr`.